packages/legacy/core/App/utils/oca.ts

```diff
@@ -17,14 +17,16 @@ export const buildFieldsFromAnonCredsCredential = (credential: CredentialExchang
 
 export const buildFieldsFromW3cCredsCredential = (value: W3cCredentialDisplay): Array<Field> => {
   return (
-    Object.entries(value.attributes).map(
-      ([key, value]) =>
-        new Attribute({
-          name: key,
-          value: value as string | number | null,
-          mimeType: typeof value === 'number' ? 'text/number' : 'text/plain',
-        })
-    ) || []
+    Object.entries(value.attributes)
+      .filter(([key]) => key !== 'id')
```

Comment:
I remember there being some potential issue with using `id` as an attribute name, but could you refresh my memory? Why are we filtering it out here?

Reply:
I think this will be Ok. id is a reserved word in W3C credentials so it looks like it's filtering out all the keys except for the reserved ones. The problem you mention is that Credo / AnonCreds does not like it when id is used as a property name. Maybe it's getting confused 🤷🏻‍♂️